src/sentry/integrations/pipeline.py

```diff
@@ -78,17 +80,30 @@ def _finish_pipeline(self, data):
             if created:
```

Comment:
while you're here I think this should be `if not created`

Reply:
Fixing in a separate PR

Reply:
https://github.com/getsentry/sentry/pull/8630